src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -199,34 +199,19 @@ object EasyIngest {
       request = (datastreamId, sdo.listFiles.find(_.getName == dsSpec.contentFile)) match {
         case ("EMD", Some(file)) =>
           // Note that this would change the ingested file's checksum, but it is only for the EMD datastream, which has no checksum
-          return executeAddRequestWithReplacement(request, file, PLACEHOLDER_FOR_DMO_ID, pidDictionary(sdo.getName))
+          request.content(replacePlaceHolder(file, PLACEHOLDER_FOR_DMO_ID, pidDictionary(sdo.getName)))
```

Comment:
For a more generic solution, stage-dataset should put something like `{{objectSDO=dataset}}` in the EMD so we can lookup the value after the '=' in the pidDcitionary. I don't think we should apply this substitution in EASY_FILE data streams. So we are still stuck with easy-knowledge we want to avoid. Any other exceptions?